e2e-tests/global-setup.js

```diff
@@ -6,6 +6,15 @@
 import { chromium, test as baseTest } from '@playwright/test';
 import { checkEnv } from './helpers/general.js';
 import { LoginPage } from './admin/pages/login.js';
+import { fileURLToPath } from 'url';
+import path from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+// Import environment variables from .env file if available
+import dotenv from 'dotenv';
```

Comment:
None blocking commnet: we usually have the `imports` block first thing on files, is there a reason for this one to be out of the import block?